x-pack/plugins/cloud_security_posture/public/pages/findings/types.ts

```diff
@@ -29,11 +30,11 @@ export interface FindingsBaseEsQuery {
 // TODO: this needs to be defined in a versioned schema
 export interface CspFinding {
   '@timestamp': string;
-  cycle_id: string;
```

Comment:
can you clarify why we need to remove this property?

Reply:
Thanks! btw the full discussion is [here](https://github.com/elastic/security-team/issues/3782)